### ISSUE TITLE
Speaker notes on remote

### ISSUE THREAD

**Issue by nmattia** [deckgo/deckdeckgo#191]
It’d be ultra sweet to have speaker notes on the phone while presenting

**Comment by peterpeterparker** [comment]
U mean in studio or in the core/developer version?

If studio sure

If core/developer it's already there, implemented in both core and remote app

**Comment by peterpeterparker** [comment]
A propos, in each slides' doc: https://docs.deckdeckgo.com/slides/title#app-slide-title-notes

```
<deckgo-deck>
  <deckgo-slide-title>
    <h1 slot="title">My presentation title</h1>
    <div slot="notes">A note regarding this particular slide</div>

And another note on a new line about it too.
  </deckgo-slide-title>
</deckgo-deck>
```

**Comment by peterpeterparker** [comment]
Solved in #440
